plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/Instrumentable.kt

```diff
@@ -0,0 +1,21 @@
+package io.sentry.android.gradle.instrumentation
+
+import io.sentry.android.gradle.instrumentation.database.AndroidXSQLiteClassVisitor
+import org.objectweb.asm.ClassVisitor
+
+enum class Instrumentable(val fqName: String) {
+    ANDROIDX_SQLITE_DATABASE("androidx.sqlite.db.framework.FrameworkSQLiteDatabase");
```

Comment:
does the instrumentation happen before proguard/r8? wondering if the class would be renamed if so

Reply:
That's a good point, I also have it in my to-do to test it with r8, thought to postpone it for later, but will check it tomorrow.

Reply:
btw whats `fq` stands for in `fqName`? maybe a more descriptive name?

Reply:
tested with r8 - it worked. The transformation step runs as part of the `minifyWithR8` gradle task, and it internally makes sure to run custom transformations before the obfuscation

> btw whats `fq` stands for in `fqName`? maybe a more descriptive name?

fq stand for fully-qualified name and it's kind of a common standard for this -> https://www.google.com/search?q=fqname. Kotlin, Google, others are using it. But if you will, I can change that, though it will look a bit long to me